### ISSUE TITLE
Bug: Desktop notifications aren't working

### ISSUE THREAD

**Issue by retlehs** [roots/bud#918]
### Terms

- [X] I have read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
- [X] This request is not a duplicate of an existing issue
- [X] I have read the [docs](https://roots.io/docs/) and followed them (if applicable)
- [X] I have seached the [Roots Discourse](https://discourse.roots.io/) for answers and followed them (if applicable)
- [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

#### What's wrong?

Bud ships with support for desktop notifications through the roots-notifer package, but it is currently not working

### Steps To Reproduce

1. `yarn dev`
1. Trigger a build

### Expected Behavior

A desktop notification should appear on a build

### Actual Behavior

Desktop notifications are not working

### Versions

v5

**Comment by retlehs** [comment]
fixed in [5.6.0](https://github.com/roots/bud/releases/tag/v5.6.0) 🎉

**Comment by StudioRATATA** [comment]
I haven't gotten this to work in 5.6.2. I've received one message at the first run after update, but is was gone too quick to see what is was. After that no notifications. Should I change something to activate it?

**Comment by retlehs** [comment]
@StudioRATATA can you try this?

```
$ chmod u+x node_modules/@roots/bud/vendor/mac.no-index/roots-notifier.app/Contents/MacOS/roots-notifier
```

**Comment by StudioRATATA** [comment]
@retlehs Thanks Ben. It gave me a new notification. Clicking it pushed me to MacOS's notification settings. Here I was able to enable the `roots-notifier`. Now It works. I'll let it up to you if the command fixed it or I did not know I had to enable it in MacOS settings. :)

Offtopic. Could the notifications be color coded? Add red text on error, green text on success. Feels more user friendly.

Cheers
